test/server/spec.js

```diff
@@ -491,17 +513,150 @@ describe('superagentCache', function(){
       );
     });
 
+    it('Global TTL settings should be overwritten by the one calculated from the response cache-control header', function (done) {
+      superagentCache = superagentCacheModule(cache, {doQuery: true, expiration: 1});
+      count = 0;
+      superagent
+        .get('localhost:3000/count')
+        .use(superagentCache)
+        .set('Cache-Control', 'max-age=3')
+        .end(function (err, response, key){
+          expect(response).toNotBe(null);
+          expect(response.body.count).toBe(1);
+          setTimeout(function(){
```

Comment:
Why is this timeout needed? To account for cache delay? I think it can be way shorter then, right? Having a test idling for 1s is not great